addons/website/tests/test_ui.py

```diff
@@ -64,6 +64,44 @@ def test_01_attachment_website_unlink(self):
 @odoo.tests.tagged('-at_install', 'post_install')
 class TestUiHtmlEditor(HttpCaseWithUserDemo):
 
+    def test_html_editor_language(self):
+        Page = self.env['website.page']
```

Comment:
used only once, let's not create a variable (especially since you did not for res.lang which is used multiple times)
or keep it and add the same for ResLang which is used multiple time actually, better